src/config.rs

```diff
@@ -98,6 +111,42 @@ impl Config {
         }
     }
 
+    /// Populate the config with the values from parsed command line arguments.
+    /// If neither `--bless` or `--check` are provided `default_bless` is used.
+    ///
+    /// The default output conflict handling command suggests adding `--bless`
+    /// to the end of the current command.
+    pub fn with_args(&mut self, args: &Args, default_bless: bool) {
+        let Args {
+            ref filters,
+            quiet: _,
+            check,
+            bless,
+            threads: _,
```

Comment:
you added a threads field to `Config` but didn't fill it in here (in case it is `Some` here)

Reply:
Whoops, I even did a destructure to avoid exactly that 😅